data/ext/bio/FormalParameter.rdfa

```diff
@@ -0,0 +1,40 @@
+<div>
+  <!--
+  # Schema.org Bioschemas extension proposal
+  * | Project: https://bioschemas.org
+  * | MailingList: https://lists.w3.org/Archives/Public/public-bioschemas/
+  * | Issues:
+  -->
+
+  <div typeof="rdfs:Class" resource="http://schema.org/FormalParameter">
+      <link property="http://schema.org/isPartOf" href="http://bio.schema.org" />
+      <span class="h" property="rdfs:label">FormalParameter</span>
+      <span property="rdfs:comment">A formal parameter is a slot that may be satisfied when the workflow is run. It appears as an Input or Output of a Workflow</span>
```

Comment:
Is there any chance to extend this to, for instance, software?

Reply:
Yes, perhaps `input` and `output` to a `FormalParameter` also make sense on a `SoftwareApplication` - (considering it could be a pretend subclass `CommandLineTool`)

..but we found it hard to put it on `SoftwareSourceCode` without adding an intermediate `Script` - what is the formal parameters for `code.c` ? Would need to break into functions etc..! If someone adds `Function` again they can reuse `FormalParameter`.

So that was one of the motivation for making a new subclass `Workflow` rather than just profiling `SoftwareSourceCode`.

Reply:
Creating a new type Workflow rather than profiling SoftwareSourceCode is well-motivated. However, the definition and description of FormalParameter could be more generic so it can later be integrated via a new property to SoftwareSourceCode

Reply:
Are you suggesting renaming it to some kind of `ExpectedValue` wrapper or just using these kind of words without mentioning "Workflow"?

Reply:
I am suggesting to take more compatible cases, such as the software one, into account when defining the type and corresponding descriptions